src/com/workspace/utill/Logger.java

```diff
@@ -0,0 +1,21 @@
+package com.workspace.utill;
+
+public class Logger {
```

Comment:
There should be only 1 entry point for the client to log the logs. but I see a couple of places here,  the client can get access to the intermediate level classes to log.